storybookjs/storybook
# Fix webpack overriding && Add an example with local file dependencies

Issue: -can't find it-

## What I did
Fixed a few prepublish stuff
Fixed the babelconfig so it loads presets from included 'node_modules'

## How to test
- `npm install`
- `cd examples/test-cra`
- `npm run storybook`

This should result in a successful storybook being launched, and no messages in the terminal about presets not being loaded.
